public/pages/HistoricalDetectorList/components/EmptyHistoricalDetectorMessage/EmptyHistoricalDetectorMessage.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { EuiButton, EuiEmptyPrompt, EuiText } from '@elastic/eui';
+import React from 'react';
+import { APP_PATH, PLUGIN_NAME } from '../../../../utils/constants';
+
+const filterText =
+  'There are no historical detectors matching your applied filters. Reset your filters to view all historical detectors.';
+const emptyText =
+  'Historical detectors allow you to detect anomalies on your historical data. You may also use them to pre-train or tune your model by using your index history to train an anomaly detector.';
```

Comment:
"index history" -> "history index" ?

Reply:
Copied from mockup. I think the idea of "index history" here is to use the historical portion of the index to create historical detectors and tune them, to help create a good configuration for a realtime detector that's using the same index.

I actually feel that this entire sentence can be removed, since we currently can't even use these historical detector pre-trained models in a real-time context. How do you feel about removing and just including the first "Historical detectors allow you to detect anomalies on your historical data" sentence? Will bring up with Carlos too.

Reply:
Got it. May need tech writer to help review.
Agree to remove the pre-train model part. We don't support this in first release.

Reply:
Actually, just checked the current mockup, looks like the text has changed to just "Use historical detectors to detect anomalies on a selected time range of your historic data." Will change to this.